std/zig/render.zig

```diff
@@ -724,15 +724,21 @@ fn renderExpression(
                             expr_widths[i] = width;
                         }
 
-                        const new_indent = indent + indent_delta;
+                        var new_indent = indent + indent_delta;
                         try renderToken(tree, stream, lbrace, new_indent, start_col, Space.Newline);
-                        try stream.writeByteNTimes(' ', new_indent);
+
+                        if (tree.tokens.at(lbrace + 1).id != Token.Id.MultilineStringLiteralLine) {
```

Comment:
Just double checking, this intentionally avoids `tree.nextToken(lbrace)` which would skip over comments, right?

Reply:
Uhm, well actually I added this, because it was adding the additional whitespace as shown in #2456. 
I think we should allow comments after the opening brackets too (which also works when no multi line string literals are used), like this:
```
const foo = [][]const u8{ // comment goes here
    \\aaa
    \\etc
, // and maybe here too?
    \\bbb
    \\
};
```
- In that case I should have used `tree.nextToken(lbrace)`, but that causes other problems, because no newline is added in the `renderToken` function when the next token is a multi line string.
- The code should be similar to the function formatting shown in #2456, but because the array initializer also supports the column format it works completely different.